group-coordinator/src/main/resources/common/message/ConsumerGroupPartitionMetadataValue.json

```diff
@@ -0,0 +1,30 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// KIP-848 is in development. This schema is subject to non-backwards-compatible changes.
+{
+  "type": "data",
+  "name": "ConsumerGroupPartitionMetadataValue",
+  "validVersions": "0",
+  "flexibleVersions": "0+",
+  "fields": [
+    { "name": "Epoch", "versions": "0+", "type": "int32" },
+    { "name": "TopicPartitionMetadata", "versions": "0+",
+      "type": "[]TopicPartition", "fields": [
```

Comment:
Here `[]TopicPartition` is used for wrapping a `uuid` and an `int32`. [Further down](https://github.com/apache/kafka/pull/13203/files#diff-61c0353c238f00c48dbf08404b1bfa18015faaa64f44e71aa9dc67d66654b17aR25) the same type is used a `uuid` and an `int32[]`. I know these types will mostly be used in generated code but I wonder if trying to use different names would help when people have to go digging in the lowest layers.

Also should we add descriptions to all the fields? Even if it's not part of the public API, and many fields are self explanatory, I think it would help the next person that has to touch this. WDYT?

Reply:
Let me see what I can do for the naming. Regarding the doc, I agree. I will add them.

Reply:
Renamed that field and added doc to add fields.